### DISCUSSION TITLE
Possible to not intercept (filter) static files?

### DISCUSSION THREAD

**Discussion by payloadartist** [google/tamperchrome#191] [Ideas]
Hi,

Is there a way to filter (not intercept) static files like css, js, svg, png, etc? 

Would be great if an (optional) filter for static files could be implemented, so that it doesn't intercept them.

Cheers

**Comment by sirdarckcat** [comment]
there's a "type" column that isn't yet implemented, but that provides information about the "sink" of the resource. https://chromedevtools.github.io/devtools-protocol/tot/Network/#type-ResourceType

```
Document, Stylesheet, Image, Media, Font, Script, TextTrack, XHR, Fetch, EventSource, WebSocket, Manifest, SignedExchange, Ping, CSPViolationReport, Preflight, Other
```

I think you would implement what you suggested with a filter like XHR/Fetch.

**Reply by sirdarckcat** [reply]
Follow this on #196 and #197
